salt/iiif/config/opt-loris-docker-compose.yaml

```diff
@@ -7,6 +7,9 @@ services:
             NEW_RELIC_ENABLED: "{{ pillar.elife.newrelic.enabled }}"
         logging: # https://docs.docker.com/compose/compose-file/#logging
             driver: journald
+            options:
+                # use 'uwsgi' instead of (obscure) process ID
+                tag: "uwsgi"
```

Comment:
oh that's the trick, good to know. It's probably obvious that `uwsgi` is Loris as there is only one application running on the `iiif` servers.

Reply:
you can actually do some clever things with it as it supports templated strings as well (that conflict with Salt+Jinja). I used 'uwsgi' because it's the uwsgi log output in the uwsgi log format.